src/lein_monolith/plugin.clj

```diff
@@ -4,76 +4,93 @@
   (:require
     [lein-monolith.config :as config]
     [lein-monolith.dependency :as dep]
+    [leiningen.core.eval :as eval]
     [leiningen.core.main :as lein]
     [leiningen.core.project :as project]))
 
-
 ;; ## Profile Generation
 
+(def profile-config
+  "Configuration for inherited profiles. Structured as a vector of pairs to
+  maintain ordering."
```

Comment:
Could you explain why ordering is significant?

Reply:
I've expanded the comment.